File: file_processing/py/filter_evens.py
#!/usr/bin/python
## print even numbers in file in sys.argv[1]
import sys
file_path = sys.argv[1]
with open(file_path, 'r') as infile:
    for n in [int(x) for x in infile.readlines() if int(x) % 2 == 0]:
        print n

File: file_processing/pl/filter_evens.pl
#!/usr/bin/perl
## print event numbers from file in $ARGV[0]
use strict;
use warnings;
my $file_path = shift();
open(my $infile, '<', $file_path) or die $!;
while ( <$infile> ) {
    print $_ if ( $_ % 2 == 0 );
}
